cnv/cnv_users_guide/cnv-using-the-default-pod-network-with-cnv.adoc

```diff
@@ -24,7 +24,7 @@ select the required binding method from the *Networking* screen.
 
 include::modules/cnv-networking-wizard-fields-web.adoc[leveloffset=+2]
 
-== Configuration file examples
+== Virtual machine configuration templates for the default network
```

Comment:
CNV has virtual machine template objects. To avoid confusion, I think we should avoid using 'template' to describe example configurations. (However this is an issue larger than this one line, and we need a separate PR to change all instances for consistency, so this is not a blocker.)

Reply:
I can change it to "Virtual machine configuration examples for the default network". Do you want me to create a Jira story about the larger issue? Thanks for reviewing @aburdenthehand.

Bob

Reply:
@bgaydosrh - works for me :)
Jira is a good idea too. Thanks Bob.

Reply:
Done, @aburdenthehand. I created https://jira.coreos.com/browse/CNV-3458 which I can work  in Sprint 176. I have assigned it low priority and a non-blocker.